depthai_nodes/ml/parsers/yunet.py

```diff
@@ -128,6 +127,32 @@ def setOutputLayerNames(
         self.conf_output_layer_name = conf_output_layer_name
         self.iou_output_layer_name = iou_output_layer_name
 
+    def setInputShape(self, width, height):
```

Comment:
These two functions are dobled. Probably a mistake in merging changes from other commits?

Reply:
Yeap, thanks for noticing that. I am deleting the doubled functions.